src/EditorFeatures/Test/MoveType/AbstractMoveTypeTest.cs

```diff
@@ -96,6 +97,12 @@ protected override CodeRefactoringProvider CreateCodeRefactoringProvider(Workspa
                     // the original source document does not exist in the new solution.
                     var newSolution = oldSolutionAndNewSolution.Item2;
                     Assert.Null(newSolution.GetDocument(oldDocumentId));
+
```

Comment:
Looks like we're not asserting that the file rename actually happened :( . It should, given that this is `TestRenameFileToMatchTypeAsync`. If its simple, could you add an assert for that -- Thanks!